spec/features/article_spec.rb

```diff
@@ -0,0 +1,53 @@
+require "rails_helper"
+
+describe "article", :type => :feature do
+  it "successfully creates a new article" do
+    visit '/articles/new'
+    within('#new_article') do
+      fill_in 'article_title', :with => 'This is an article'
+      fill_in 'article_text', :with => 'This is the articles body'
+    end
+
+    click_button 'Create Article'
+    expect(page).to have_content 'Articles'
```

Comment:
you also expect that page to contain something about the article, either the title or the text